build_projects/dotnet-cli-build/CheckIfAllBuildsHavePublished.cs

```diff
@@ -71,7 +71,7 @@ public override bool Execute()
                 string name = Path.GetFileName(file);
                 foreach (string img in badges.Keys)
                 {
-                    if ((name.StartsWith($"{img}")) && (name.EndsWith(".svg")))
+                    if ((name.StartsWith($"{string.Concat(img, "_")}")) && (name.EndsWith(".svg")))
```

Comment:
No need to call string.concat and interpolate. Pick one 

`string.Concat(img, "_")`
`$"{img}_"`

Or, for completeness
`img + "_"`